overrides/app/pages/algolia-product-detail/index.jsx

```diff
@@ -240,17 +250,15 @@ const ProductDetail = () => {
                 quantity
             }))
 
-            await addItemToBasketMutation.mutateAsync({
-                parameters: {basketId: basket.basketId},
-                body: productItems
-            })
+            await addItemToNewOrExistingBasket(productItems)
 
             einstein.sendAddToCart(productItems)
 
             // If the items were successfully added, set the return value to be used
             // by the add to cart modal.
             return productSelectionValues
         } catch (error) {
+            console.log('error', error)
```

Comment:
Don't you want to use the logger here too?

Reply:
We really changed a small part in product detail page and this line comes from [original `retail-react-app`: ](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/0e7684563112e9121251ea9af4f0595c012c6c8f/packages/template-retail-react-app/app/pages/product-detail/index.jsx#L310)